src/components/Send.jsx

```diff
@@ -1,17 +1,103 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import { useState } from 'react'
 import { useLocation } from 'react-router-dom'
 import * as rb from 'react-bootstrap'
-import { serialize, ACCOUNTS } from '../utils'
+import PageTitle from './PageTitle'
+import ToggleSwitch from './ToggleSwitch'
+import { useCurrentWalletInfo, useSetCurrentWalletInfo, useCurrentWallet } from '../context/WalletContext'
+import { useSettings } from '../context/SettingsContext'
+import { serialize } from '../utils'
 import * as Api from '../libs/JmWalletApi'
 
-export default function Payment({ currentWallet }) {
+const CollaboratorsSelector = ({ numCollaborators, setNumCollaborators }) => {
+  const [usesCustomNumCollaborators, setUsesCustomNumCollaborators] = useState(false)
+
+  const collaboratorsSelection = [3, 5, 6, 9]
+
+  return (
+    <rb.Form.Group className="collaborators-selector">
+      <rb.Form.Label className="mb-0">Choose a number of collaborators ({numCollaborators})</rb.Form.Label>
+      <div className="mb-2">
+        <rb.Form.Text className="text-secondary">
+          A higher number is better for privacy, but also increases the fee.
+        </rb.Form.Text>
+      </div>
+      <div className="d-flex flex-row flex-wrap">
+        {collaboratorsSelection.map((number, index) => {
+          return (
+            <rb.Button
+              key={index}
+              variant="white"
```

Comment:
```suggestion
              variant={settings.theme === 'light' ? 'white' : 'dark'}
```

Content of the buttons are not visible in dark mode. This fixes the visibility for now, but borders are still not quite right in dark mode.

Reply:
:+1: